File: utils/Character.py
from dataclasses import dataclass

@dataclass
class Character:
    character_id: int
    name: str
    picture_url: str
    votes_for: int
    votes_against: int

if __name__ == '__main__':
    char = Character(100, "iron man", "www.xd.com", 100, 0)
    print(char)
    print(char.__dict__)

File: demo/lib/models/Character.dart
import 'package:demo/dataAccess/CharactersDataAccess.dart';

class Character {
  String documentId;
  int characterId;
  String name;
  String imgUrl;
  int votesFor;
  int votesAgainst;

  Character(this.documentId, this.characterId, this.name, this.imgUrl,
      this.votesFor, this.votesAgainst);

  void getVoteFor() {
    votesFor += 1;
    increaseVoteFor(documentId, votesFor);
  }

  void getVoteAgainst() {
    votesAgainst += 1;
    increaseVoteAgainst(documentId, votesAgainst);
  }
}
